### PR TITLE
2808 convert to es6 classes

### PR DESCRIPTION
**Pull request by eleanor-heath**
https://github.com/adaptlearning/adapt_framework/issues/2808

### CODE REVIEW

File: js/hotgraphicView.js [hunk 1]
```diff
@@ -1,171 +1,165 @@
...
-        'change:_isVisited': this.onItemsVisitedChange
-      });
-    }
+  setUpModelData() {
+    if (this.model.get('_canCycleThroughPagination') !== undefined) return;
+    this.model.set('_canCycleThroughPagination', false);
+  }
 
-    reRender() {
-      if (Adapt.device.screenSize === 'large' || this.model.get('_isNarrativeOnMobile') === false) return;
```
Comment (cahirodoherty-learningpool) [inline js/hotgraphicView.js:44]: Do we still need the _isNarrativeOnMobile check?

**Review by ethan-lp — APPROVED** [review]

**Review by oliverfoster — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by lc-alexanderbenesch — APPROVED** [review]
👀👍

File: js/hotgraphicPopupView.js [hunk 2]
```diff
@@ -1,144 +1,140 @@
...
+    const labelText = Handlebars.compile(template)({
+      itemNumber: this.model.getActiveItem().get('_index') + 1,
+      totalItems: this.model.get('_items').length
+    });
+    this.$('.hotgraphic-popup__count').html(labelText);
+  }
+
+  handleTabs() {
+    Adapt.a11y.toggleHidden(this.$('.hotgraphic-popup__item:not(.is-active) *'), true);
+    Adapt.a11y.toggleHidden(this.$('.hotgraphic-popup__item.is-active *'), false);
```
Comment (oliverfoster) [inline js/hotgraphicPopupView.js:61]: Why have these been added?

**Pull request merged**